Parsers/ASimAuthentication/Parsers/ASimAuthenticationCiscoASA.yaml

```diff
@@ -0,0 +1,224 @@
+Parser:
+  Title: Authentication ASIM parser for Cisco Device Logon Events
+  Version: '0.1.0'
+  LastUpdated: 19 Jul 2023
+Product:
+  Name: Cisco Adaptive Security Appliance (ASA)
+Normalization:
+  Schema: Authentication
+  Version: '0.1.3'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/ASimAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: About Cisco ASA Messages
+  Link: https://www.cisco.com/c/en/us/td/docs/security/asa/syslog/b_syslog/about.html
```

Comment:
can we add below link too?
https://www.cisco.com/c/en/us/td/docs/security/asa/syslog/b_syslog/syslogs-sev-level.html

Reply:
Done